test/test_expanded_weights.py

```diff
@@ -187,10 +187,14 @@ def test_unsupported_expand_weights(self, device, dtype, op):
     def test_expanded_weight_forward(self, device, dtype, op):
         sample_inputs = op.sample_inputs(device, dtype)
         for sample_input in supported_inputs(op, sample_inputs):
+            if op.name == "nn.functional.embedding":  # embedding flips its argument order for autograd tests
+                sample_input = SampleInput(sample_input.args[0], args=(sample_input.input,), kwargs=sample_input.kwargs)
+                if 'max_norm' in sample_input.kwargs:  # in place update makes it difficult even with copying
```

Comment:
The fact that this worked on the last commit is a little suss to me because it suggests that the test that was failing (linux-xenial-cuda11.3-py3.7-gcc7) is failing because it's not copying. I'll try to make a smaller repro

Reply:
@samdow is this still failing? I'd like to understand what the problem is before this goes in (it sounds concerning)